index.js

```diff
@@ -378,11 +379,26 @@ function Argv (processArgs, cwd) {
   }
 
   var versionOpt = null
-  self.version = function (ver, opt, msg) {
-    versionOpt = opt || 'version'
-    usage.version(ver)
+  self.version = function (opt, msg, ver) {
+    if (arguments.length === 0) {
+      ver = parent(__dirname).parse().version
+      versionOpt = 'version'
+      msg = usage.deferY18NLookup
```

Comment:
this looks like a bug, should be:

```js
msg = usage.deferY18nLookup('Show version number')
```

There's starting to be some duplicate code here, I would actually refactor to something like this:

```js
self.version = function (opt, msg, ver) {
  if (arguments.length === 0) {
    ver = parent(__dirname).parse().version
    opt = 'version'
  } else if (arguments.length === 1) {
    ver = opt
    opt = 'version'
  } else if (arguments.length === 2) {
    ver = msg
  }

  versionOpt = opt
  msg = msg || usage.deferY18nLookup('Show version number')

  usage.version(ver)
  self.boolean(versionOpt)
  self.describe(versionOpt, msg)
  return self
}
```